### ISSUE TITLE
Ownable: ConcurrentModificationException when removing a player's owned fields

### ISSUE THREAD

**Issue by algorythm** [hold11/CDIO_del3#77]
When trying to remove a player's owned fields (on bankruptcy), the game crashes. I suspect it is due to the fact, that the field was never added to the `ownedOwnalbes`-list in `Ownable`-class.

This issue was found while solving #74. When a player goes bankrupt, the player object gets removed from the player-list and their owned fields gets reset to not have an owner. This results in removing the `Ownable` from the `ownedOwnables`-list, which propably does not even exist.

**Comment by algorythm** [comment]
```
--------------- Player 2 ---------------
Player 2's turn. Current balance: 4175
Player 2 rolled 1 and 3 (4).
Player 2 landed on The Buccaneers
Player 2's balance is now 4175

Current Balance: 4175 | Plot Price: 4000
This plot is not owned, do you want to buy? yes
Player 2 just bought The Buccaneers for 4000.
Player 2's balance is now 175.


--------------- Player 1 ---------------
Player 1's turn. Current balance: 5130
Player 1 rolled 4 and 3 (7).
Player 1 landed on Sea Grover
Player 1's balance is now 5130

Current Balance: 5130 | Plot Price: 4000
This plot is not owned, do you want to buy? yes
Exception in thread "main" java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:901)
	at java.util.ArrayList$Itr.next(ArrayList.java:851)
	at fields.Ownable.removePlayersOwnables(Ownable.java:69)
	at models.Player.removePlayer(Player.java:145)
	at models.GameLogic.checkBankruptcy(GameLogic.java:212)
	at models.GameLogic.playTurn(GameLogic.java:58)
	at Main.main(Main.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Player 1 just bought Sea Grover for 4000.
Player 1's balance is now 1130.


--------------- Player 2 ---------------
Player 2's turn. Current balance: 175
Player 2 rolled 4 and 6 (10).
Player 2 landed on Palace Gates
Player 2's balance is now 0

Process finished with exit code 1
```

**Comment by algorythm** [comment]
Fixed in [version 1.2.2](https://github.com/hold11/CDIO_del3/releases/tag/v1.2.2).
